### PR TITLE
Add a set of new python wrappers

### PR DESCRIPTION
**Pull request by agrayver**
Keep extending python wrappers. 

Part of #9015

### CODE REVIEW

**Comment by agrayver** [comment]
@Rombur @masterleinad I would appreciate your review here. 

Apart from new methods, I have changed the save triangulation behavour to write in binary instead of the text. Since I already use python binds for real problems, text format appeared much slower and less efficient.

File: contrib/python-bindings/source/triangulation_wrapper.cc [hunk 1]
```diff
@@ -1435,31 +1520,31 @@ namespace python
   void
   TriangulationWrapper::save(const std::string &filename) const
   {
-    std::ofstream                 ofs(filename);
-    boost::archive::text_oarchive oa(ofs);
+    std::ofstream                   ofs(filename);
+    boost::archive::binary_oarchive oa(ofs);
```
Comment (Rombur) [inline contrib/python-bindings/source/triangulation_wrapper.cc]: Any reason for this change?

Comment (agrayver) [reply contrib/python-bindings/source/triangulation_wrapper.cc]: Yes. The text format is slower and occupies much more space on the disk. This is especially apparent in 3-D (I am using this already for production runs...).

**Comment by agrayver** [comment]
@Rombur I addressed the comments.

File: contrib/python-bindings/include/cell_accessor_wrapper.h [hunk 2]
```diff
@@ -180,6 +180,47 @@ namespace python
...
 
+    /**
+     * Test whether the cell has children (this is the criterion for activity of
+     * a cell).
+     */
+    bool
+    active() const;
+
+    /**
+     * Returns the level within the mesh hierarchy at which this cell is
```
Comment (Rombur) [inline contrib/python-bindings/include/cell_accessor_wrapper.h]:
```suggestion
     * Return the level within the mesh hierarchy at which this cell is
```

File: contrib/python-bindings/source/export_triangulation.cc [hunk 3]
```diff
@@ -112,6 +112,16 @@ namespace python
 
 
 
+  const char dim_docstring[] =
+    "Return the dimension of the Triangulation                              \n";
+
+
+
+  const char spacedim_docstring[] =
+    "Return the space dimension of the Triangulation                        \n";
```
Comment (Rombur) [inline contrib/python-bindings/source/export_triangulation.cc]: Do you want to add a final dot at the end of the comment.

**Comment by agrayver** [comment]
@Rombur Done.

**Pull request merged**